CHANGELOG.md

```diff
@@ -16,6 +16,16 @@ and this project adheres to [Semantic Versioning][].
 
 - Apply default values for benchmarker metrics {pr}`203`.
 
+### Added
```

Comment:
Can you move these into the unreleased section above?

Reply:
I did! Apologies for the oversight. I thought version 0.5.4 had not been released yet, as it doesn't show as "latest" on the main page of the package on GitHub.